### DISCUSSION TITLE
Show NAN in Inspector

### DISCUSSION THREAD

**Discussion by timothyqiu** [godotengine/godot-proposals#8967] [Editor]
When a float property is NAN, it's shown as 0 in the Inspector.

It's confusing and makes debugging harder. (e.g. when Inspecting a node in Remote scene tree)

**Comment by AThousandShips** [comment]
I'd call this a bug, possibly an oversight in the implementation, haven't looked at the code though

**Reply by Calinou** [reply]
Should the same be done for INF as well?

I also don't remember if NAN and -NAN should be displayed the same in the inspector.

**Reply by AThousandShips** [reply]
The minus is just signalling/non-signalling right? Does that matter in the codebase at all? (Though the exact definition of the NANs is implementation defined)
